src/test/java/com/puppycrawl/tools/checkstyle/api/AbstractFileSetCheckTest.java

```diff
@@ -95,6 +96,7 @@ public void testProcessException() throws Exception {
         }
         catch (IllegalArgumentException ex) {
             // exception is expected
+            assertEquals("Invalid exception message", "Test", ex.getMessage());
```

Comment:
http://checkstyle.sourceforge.net/config_blocks.html#EmptyCatchBlock could have caught this.

Reply:
A comment is enough to pass the check `EmptyCatchBlock`.
PMD is more strict in this case.

Reply:
@romani What is your thoughts?

Reply:
default format is `.*` in Check.
But we can to be more strict, especially then we already upgraded code to satisfy other tools.

Reply:
FYI, PMD's EmptyCachBlock rule does allow to ignore blocks based on the name of the exception (similar to Checkstyle), by setting the property `allowExceptionNameRegex`. 6.+ versions default to `^(ignored|exception)$`. See https://pmd.github.io/pmd/pmd_rules_java_errorprone.html#emptycatchblock

Reply:
@jsotuyod There is nothing wrong with PMD's version of the rule since it was the one who caught this area.
The issue was our own check didn't catch it, but it is because it is set to ignore the exception if it empty and has a comment in it.